pkg/scalers/influxdb_scaler.go

```diff
@@ -41,11 +43,18 @@ func NewInfluxDBScaler(config *ScalerConfig) (Scaler, error) {
 	}
 
 	influxDBLog.Info("starting up influxdb client")
-
-	return &influxDBScaler{
-		client:   influxdb2.NewClient(meta.serverURL, meta.authToken),
-		metadata: meta,
-	}, nil
+	//In case unsafeSsL is enabled.
```

Comment:
```suggestion
	// In case unsafeSsL is enabled.
```
You should add space here